client/src/main/java/io/getstream/chat/android/client/notifications/handler/NotificationConfig.kt

```diff
@@ -6,9 +6,11 @@ data class NotificationConfig(
     val notificationChannelId: Int = R.string.stream_chat_notification_channel_id,
     val notificationChannelName: Int = R.string.stream_chat_notification_channel_name,
     val smallIcon: Int = R.drawable.stream_ic_notification,
-    val firebaseMessageIdKey: String = "stream-chat-message-id",
-    val firebaseChannelIdKey: String = "stream-chat-channel-id",
-    val firebaseChannelTypeKey: String = "stream-chat-channel-type",
+    val firebaseMessageIdKey: String = "message_id",
+    val firebaseMessageTextKey: String = "message_text",
+    val firebaseChannelIdKey: String = "channel_id",
+    val firebaseChannelTypeKey: String = "channel_type",
+    val firebaseChannelNameKey: String = "channel_name",
```

Comment:
Are we updating these because they changed on our backend?

Reply:
They are customizable by user, however, now those values should correspond to the default ones on dashboard